superset-frontend/src/SqlLab/components/SqlEditorLeftBar/index.tsx

```diff
@@ -17,86 +17,100 @@
  * under the License.
  */
 import React from 'react';
-import PropTypes from 'prop-types';
 import Button from 'src/components/Button';
-import { t, styled, css } from '@superset-ui/core';
+import { t, styled, css, SupersetTheme } from '@superset-ui/core';
 import Collapse from 'src/components/Collapse';
 import Icons from 'src/components/Icons';
 import TableSelector from 'src/components/TableSelector';
 import { IconTooltip } from 'src/components/IconTooltip';
-import TableElement from '../TableElement';
+import { QueryEditor } from 'src/SqlLab/types';
+import { DatabaseObject } from 'src/components/DatabaseSelector';
+import TableElement, { Table, TableElementProps } from '../TableElement';
 
-const propTypes = {
-  queryEditor: PropTypes.object.isRequired,
-  height: PropTypes.number,
-  tables: PropTypes.array,
-  actions: PropTypes.object,
-  database: PropTypes.object,
-  offline: PropTypes.bool,
-};
+interface ExtendedTable extends Table {
+  expanded: any;
```

Comment:
Is this able to be described by a type? I remember seeing this with you the other day but don't remember exactly what it was, I think this might have been a `string` or a `boolean`?

Reply:
I'll have to look at a previous commit to see what it was previously. Once I do I'll change it to that type.

Reply:
Changed the expanded type from any to boolean, which is what it was originally.👍